### ISSUE TITLE
Idea plugin print error message “ebean-enhance> cls: ...”

### ISSUE THREAD

**Issue by ooknight** [ebean-orm/ebean#1761]
There is spring-boot-starter-data-redis in my classpath and enable idea plugin.

```
2019-07-18 14:12:18.405  INFO 7508 --- [           main] org.ufox.demo.ApplicationTest            : No active profile set, falling back to default profiles: default
ebean-enhance> cls: org/springframework/data/web/PageableHandlerMethodArgumentResolver  msg: defaulted common supertype for type1:org/springframework/data/web/SortHandlerMethodArgumentResolver type2:org/springframework/data/web/SortArgumentResolver due to err:java.lang.IllegalArgumentException: java.io.IOException: Class not found
ebean-enhance> cls: org/springframework/mail/javamail/JavaMailSenderImpl  msg: defaulted common supertype for type1:java/lang/Object type2:javax/mail/internet/MimeMessage due to err:java.lang.IllegalArgumentException: java.io.IOException: Class not found
ebean-enhance> cls: org/springframework/data/redis/connection/jedis/JedisConnection  msg: defaulted common supertype for type1:redis/clients/jedis/Pipeline type2:redis/clients/jedis/Transaction due to err:java.lang.IllegalArgumentException: java.io.IOException: Class not found
ebean-enhance> cls: org/springframework/data/redis/connection/jedis/JedisConnection  msg: defaulted common supertype for type1:redis/clients/jedis/exceptions/JedisDataException type2:java/lang/Object due to err:java.lang.IllegalArgumentException: java.io.IOException: Class not found
2019-07-18 14:12:18.988  INFO 7508 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2019-07-18 14:12:18.994  INFO 7508 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-07-18 14:12:19.050  INFO 7508 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 23ms. Found 0 repository interfaces.
2019-07-18 14:12:19.606  INFO 7508 --- [           main] io.ebean.EbeanVersion                    : ebean version: 11.41.2
2019-07-18 14:12:19.655  INFO 7508 --- [           main] io.ebean.config.properties.LoadContext   : loaded properties from [application.properties]
2019-07-18 14:12:19.686  INFO 7508 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-07-18 14:12:19.884  INFO 7508 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2019-07-18 14:12:19.888  WARN 7508 --- [           main] io.ebean.internal.DefaultContainer       : DataSource [db] has autoCommit defaulting to true!
2019-07-18 14:12:19.906  INFO 7508 --- [           main] io.ebean.internal.DefaultContainer       : DatabasePlatform name:db platform:h2
ebean-enhance> cls: reactor/core/publisher/FluxMetricsFuseable  msg: defaulted common supertype for type1:io/micrometer/core/instrument/MeterRegistry type2:io/micrometer/core/instrument/composite/CompositeMeterRegistry due to err:java.lang.IllegalArgumentException: java.io.IOException: Class not found
2019-07-18 14:12:21.014  INFO 7508 --- [           main] org.ufox.demo.ApplicationTest            : Started ApplicationTest in 2.968 seconds (JVM running for 4.033)
io.ebeaninternal.server.core.DefaultServer@52354202
2019-07-18 14:12:21.364  INFO 7508 --- [       Thread-4] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2019-07-18 14:12:21.367  INFO 7508 --- [       Thread-4] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
```

**Comment by rbygrave** [comment]
Yes this output is an error.  It doesn't break anything (so we can ignore the `ebean-enhance>` messages for now).

It is fixed by https://github.com/ebean-orm/ebean-agent/issues/121 - ebean-agent 11.41.3

There is a mechanism in the Idea plugin where we can 
- `mkdir ~/.ebean`
- `cp ~/.m2/repository/io/ebean/ebean-agent/11.41.3/ebean-agent-11.41.3.jar ~/.ebean`

Then in Idea toggle the enhancement off and then back on.  When the plugin is toggled on it looks for the agent to use first in the `~/.ebean` directory.  It there is a ebean-agent-x.jar in that directory then it will be used - otherwise it will use the ebean-agent that comes with the Idea plugin itself.

So you can try doing that (to use the newly released ebean-agent 11.41.3) .. or wait for the 
updated plugin but that typically takes 2 days to get approved by Jetbrains.

Let me know if any of that isn't clear.

Cheers, Rob.

**Comment by rbygrave** [comment]
I have released the Idea plugin to Jetbrains.  It will take 2 days to get approved etc.  

I'll close this for now. Confirm the fix with the new plugin in a few days or with the instructions above to use the 11.41.3 version of ebean-agent.

Cheers, Rob.
